src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -869,7 +869,7 @@ private void testGetFeedbackSessionResultsForUser() throws Exception {
         actualStrings.add(nobodyUserName);
 
         // case: Anonymous User
-        String anonymousUserName = "Anonymous " + System.currentTimeMillis();
+        String anonymousUserName = Const.DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT + System.currentTimeMillis();
```

Comment:
Add the whitespace back on this line. Although the tests seem to be passing with this change, we should avoid modifying behaviour if possible.